workshops/templates/workshops/training_progresses_inline.html

```diff
@@ -0,0 +1,17 @@
+{% load training_progress %}
```

Comment:
I believe that the `training_progress` template tag can be removed from `all_trainees.html`